src/index.js

```diff
@@ -50,6 +51,15 @@ const isReactComponent = (value: any) =>
   value && value.$$typeof === reactElement;
 
 function diffStrings(valueA: any, valueB: any, options: Options) {
+  const multiline =
+    MULTILINE_REGEXP.test(valueA) && MULTILINE_REGEXP.test(valueB);
+
+  // jest-diff doesn't support single-line values, but we want to
+  if (typeof valueA === 'string' && typeof valueB === 'string' && !multiline) {
```

Comment:
If one value is single-line and other value is multi-line, does the `if` statement append newline to both?

How about separate `if` statement for each value which tests RegExp only if typeof string? Although this seems like the primary use case, from quickly reading code in `index.js` of `jest-diff` package (but not trying it, I admit :) I think the snapshot-diff assertion works for other objects, not just React elements, true?

```js
if (typeof valueA === 'string' && !MULTILINE_REGEXP.test(valueA)) {
  valueA += '\n'; // eslint-disable-line no-param-reassign
}
if (typeof valueB === 'string' && !MULTILINE_REGEXP.test(valueB)) {
  valueB += '\n'; // eslint-disable-line no-param-reassign
}
```

Reply:
Yea, makes sense, addressed